src/dialogs/voice-assistant-setup/voice-assistant-setup-step-local.ts

```diff
@@ -222,20 +237,72 @@ export class HaVoiceAssistantSetupStepLocal extends LitElement {
     }
   }
 
-  private _findLocalEntities() {
+  private readonly _ttsProviderName = "piper";
+
+  private readonly _ttsAddonName = "core_piper";
+
+  private readonly _ttsHostName = "core-piper";
+
+  private readonly _ttsPort = "10200";
+
+  private get _sttProviderName() {
+    return this.localOption === "focused_local"
+      ? "speech-to-phrase"
+      : "faster-whisper";
+  }
+
+  private get _sttAddonName() {
+    return this.localOption === "focused_local"
+      ? "core_speech-to-phrase"
+      : "core_whisper";
+  }
+
+  private get _sttHostName() {
+    return this.localOption === "focused_local"
+      ? "core-speech-to-phrase"
+      : "core-whisper";
+  }
+
+  private readonly _sttPort = "10300";
+
+  private async _findLocalEntities() {
     const wyomingEntities = Object.values(this.hass.entities).filter(
       (entity) => entity.platform === "wyoming"
     );
-    this._localTts = wyomingEntities.filter(
-      (ent) => computeDomain(ent.entity_id) === "tts"
+    if (!wyomingEntities.length) {
+      this._localStt = [];
+      this._localTts = [];
+      return;
+    }
+    const wyomingInfo = await fetchWyomingInfo(this.hass);
+
+    const entityRegs = Object.values(
+      await getExtendedEntityRegistryEntries(
+        this.hass,
+        wyomingEntities.map((ent) => ent.entity_id)
+      )
+    );
+
+    this._localTts = entityRegs.filter(
+      (ent) =>
+        computeDomain(ent.entity_id) === "tts" &&
+        ent.config_entry_id &&
+        wyomingInfo.info[ent.config_entry_id].tts.some(
```

Comment:
Check if fetchWyomingInfo.info is set before read `[ent.config_entry_id].tts`

Reply:
How can it not be set?